src/manifest/dbtProject.ts

```diff
@@ -170,6 +173,39 @@ export class DBTProject implements Disposable {
     );
   }
 
+  private createLastRunResultsWatcher() {
+    const watcher = workspace.createFileSystemWatcher(
+      new RelativePattern(this.getTargetPath()!, `run_results.json`),
+    );
+
+    watcher.onDidChange((e) => this.invalidateCacheUsingLastRun(e));
+    watcher.onDidCreate((e) => this.invalidateCacheUsingLastRun(e));
+    return watcher;
+  }
+
+  private async invalidateCacheUsingLastRun(file: Uri) {
+    const fileContent = (await workspace.fs.readFile(file)).toString();
```

Comment:
It seems that almost everywhere in our code base we use `readFileSync(file, "utf8")`. Why not use the same?